src/request/sign-transaction/SignTransaction.js

```diff
@@ -111,32 +156,25 @@ class BaseLayout {
                 this._onConfirm(request, resolve, reject, passphrase);
             },
         );
-
-        /** @type {HTMLElement} */
-        this.$accountDetails = (this.$el.querySelector('#account-details'));
-        const $accounts = this.$el.querySelectorAll('.account');
-        $accounts.forEach($item => $item.addEventListener('click', event => this._openDetails($item, event)));
-        /** @type {HTMLButtonElement} */
-        const $closeDetails = (this.$accountDetails.querySelector('#close-details'));
-        $closeDetails.addEventListener('click', this._closeDetails.bind(this));
-        /** @type {HTMLElement} */
-        const $background = (this.$el.querySelector('#background-overlay'));
-        $background.addEventListener('click', this._closeDetails.bind(this));
     }
 
     /**
-     * @param {Element} $el
-     * @param {Event} event
+     * @param {'sender' | 'recipient'} which
      */
-    _openDetails($el, event) {
-        event.preventDefault();
-        /** @type {HTMLElement} */
-        (this.$accountDetails.querySelector('#details')).innerHTML = $el.innerHTML;
-        this.$el.classList.add('open');
+    _openDetails(which) {
+        const $el = this.$accountDetails.querySelector(`#details > .${which}`);
+        if ($el) {
+            $el.classList.remove('display-none');
+        }
+        this.$el.classList.add('account-details-open');
```

Comment:
Why are both `display-none` and `account-details-open` relevant? Shouldn't one class for the active state be enough and it be hidden for the non-class case?

Reply:
`.display-none` determines which one will be shown: recipient or sender. `account-details-open` opens the overlay.